### PR TITLE
e2e test changes to support china regions

### PR DESCRIPTION
**Pull request by cgchinmay**
### Issue
NA

### Description
Cannot run e2e tests in china region. The color teller application image is region specific so running e2e tests in china fails to pull this image. So moved this image to public docker registry so that it can be pulled in china region. 

### Checklist
- [ ] Added tests that cover your change (if possible)
- [ ] Added/modified documentation as required (such as the `README.md`, or the `docs` directory)
- [ ] Manually tested
- [ ] Made sure the title of the PR is a good description that can go into the release notes

### BONUS POINTS checklist: complete for good vibes and maybe prizes?! :exploding_head:
- [ ] Backfilled missing tests for code in same general area :tada:
- [ ] Refactored something and made the world a better place :star2:

### CODE REVIEW

File: test/framework/utils/constants.go [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package utils
+
+const (
+	COLORTELLER_IMAGE = "chinmay5j/colorteller:v1"
```
Comment (kishorj) [inline test/framework/utils/constants.go:4]: ControllerName for the const name

Comment (kishorj) [reply test/framework/utils/constants.go:4]: also why not use the tag latest?

Comment (cgchinmay) [reply test/framework/utils/constants.go:4]: just to know exact version if anything changes on the image, so usually I don't use latest but in this case I can change it.

File: test/framework/manifest/fixed_response_service_builder.go [hunk 2]
```diff
@@ -93,7 +95,7 @@ func (b *fixedResponseServiceBuilder) buildDeployment(namespace string, name str
 					Containers: []corev1.Container{
 						{
 							Name:  "app",
-							Image: "970805265562.dkr.ecr.us-west-2.amazonaws.com/colorteller:latest",
+							Image: utils.COLORTELLER_IMAGE,
```
[the remark this answers is not in the document]
Comment (cgchinmay) [reply test/framework/manifest/fixed_response_service_builder.go]: I did run all tests under e2e, didn't have any issues

**Pull request merged**